lib/bundler/fetcher.rb

```diff
@@ -6,6 +6,7 @@ module Bundler
 
   # Handles all the fetching with the rubygems server
   class Fetcher
+    autoload :CompactIndex, 'bundler/fetcher/compact_index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.